agents/ls-typescript/src/main/resources/org.eclipse.che.ls.typescript.script.sh

```diff
@@ -26,7 +26,7 @@ unset PACKAGES
 command -v tar >/dev/null 2>&1 || { PACKAGES=${PACKAGES}" tar"; }
 command -v curl >/dev/null 2>&1 || { PACKAGES=${PACKAGES}" curl"; }
 
-AGENT_BINARIES_URI=https://codenvy.com/update/repository/public/download/org.eclipse.che.ls.typescript.binaries
+AGENT_BINARIES_URI=https://codenvy.com/update/repository/public/download/org.eclipse.che.ls.typescript.binaries/1.0.1
```

Comment:
Can you explain why `ls` agents binaries should be downloaded from `codenvy.com` while `ws-agent`, `terminal` etc will be mounted or downloaded from [ws-master](https://github.com/eclipse/che/blob/CHE-5431/wsagent/agent/src/main/resources/org.eclipse.che.ws-agent.script.sh#L39)?
As far as I understand only the latest versions of `ws-master`, `terminal`, etc will be available. Can you confirm that it is OK?